src/Arcus.Testing.Tests.Unit/Assert_/AssertJsonTests.cs

```diff
@@ -0,0 +1,382 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Arcus.Testing.Tests.Unit.Assert_.Fixture;
+using Bogus;
+using FsCheck.Xunit;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Arcus.Testing.Tests.Unit.Assert_
```

Comment:
Is the underscore wanted here?

Reply:
Yeah, as to not conflict with xUnit's `Assert` class. 😉

Reply:
I'll merge this, but we can later do an additional round of checks to see what's missing. 👍